requirements.py

```diff
@@ -74,6 +83,13 @@ def parse_args(*args, **kwargs):
     return args
 
 
+def open_file(filename_arg, default):
```

Comment:
Why not use `kwargs` here? Also, inconsistent between `open()` and `.open()`?

Reply:
`.open()` is Pathlib's https://docs.python.org/3/library/pathlib.html#pathlib.Path.open
I'm not sure what you mean by `kwargs`, did you mean adding `*args, **kwargs` to the function and the open calls? Sure, I can do that but we don't currently need that.

Reply:
I prefer to use pathlib to find a sibling file in the script's directory instead of string concat.

Reply:
Disregard the kwargs comment. Why not use built-in `open` for both versus pathlib's .open?

Reply:
I was unaware you can call builtin `open` with a `Path` object, I will change that now